src/recipe/recipe.controller.ts

```diff
@@ -43,10 +44,12 @@ export class RecipeController {
   @ApiUnauthorizedResponse({ description: 'User is unauthorized' })
   @Post()
   async createRecipe(
+    @Request() userIdObject: { user: { id: number } },
```

Comment:
I would add some kind of custom decorator for retrieving authorized user object from the request.
Keep in mind that `@Request` will save the entire request in `userIdObject` while you only need the `userId`